.github/workflows/tests.yml

```diff
@@ -71,12 +71,13 @@ jobs:
           toolchain: ${{ matrix.rust-version }}
           override: true
 
-      - name: Start PostgreSQL
-        uses: ./.github/actions/setup-postgres
+      - name: Setup PostgreSQL
+        uses: ikalnytskyi/action-setup-postgres@v1
+        id: postgres
 
       - name: Set ROCKET_DATABASE_URL
         run: |
-          echo "ROCKET_DATABASE_URL=postgresql://postgres:postgres@localhost/postgres" >> $GITHUB_ENV
+          echo "ROCKET_DATABASE_URL=${{ steps.postgres.outputs.connection-uri }}" >> $GITHUB_ENV
```

Comment:
**n.b.** Connection URI is now returned by the action which is great since there's no need to know default credentials now.